techmatters/flex-plugins
# Chi 1644 resources redux

Review after #1180 is merged

## Description

* Adds redux layer for resources
* Adds resources container component
* Adds view resources component
* Links view resources component to stubbed resources service via redux

TODO: The side bar icon is a placeholder, we should replace it once I can get hold of the real one

### Checklist

### Related Issues
CHI-1644

### Verification steps

* Set `resourceBaseUrl` to any URL (or REACT_RESOURCE_BASE_URL env var locally) to enable the resources feature for the account
* An extra side bar menu option should appear for resources
* Clicking should take you to a resource view page showing a stubbed out resource
